Go Lint: avoid using underscores in Go names

https://github.com/yuri-norwood/bfk/blob/0fd9c62b3171f745ca205f56b40bdf858df60733/tape.go#L4

Fix `go lint`:
```
tape.go:4:2: don't use underscores in Go names; var tape_size should be tapeSize
```

--- a/tape.go
+++ b/tape.go
@@ -1,7 +1,7 @@
 package bfk
 
 var (
-	tape_size int64 = 30000
+	tapeSize int64 = 30000
 )
 
 type tape struct {
@@ -22,7 +22,7 @@ func (t *tape) Right() {
 	t.pointer++
 
 	// Check maximum not exceeded
-	if t.pointer >= tape_size {
+	if t.pointer >= tapeSize {
 		t.pointer = 0
 	}
 }
@@ -33,7 +33,7 @@ func (t *tape) Left() {
 
 	// Check minimum not exceeded
 	if t.pointer < 0 {
-		t.pointer = tape_size - 1
+		t.pointer = tapeSize - 1
 	}
 }
 

